### PR TITLE
[GR-38665] Avoid compile failures on AArch64 when full infopoints enabled

### PR DESCRIPTION
**Pull request by adinn**
This patch fixes the problem reported in issue #4542.

### CODE REVIEW

**Comment by adinn** [comment]
@teshull @olpaw Could one of you please review this patch and see it through the Oracle gate?

I tested this on AArch64 by modifying class SVMHost as described in the issue and it avoids the compile failures as expected.

Proper testing will need to wait for the debuginfo lcoal vars patch to be included at which point enabling debug info will also enable full infopoint generation.

File: substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/CalleeSavedRegisters.java [hunk 1]
```diff
@@ -100,6 +100,9 @@ public void verifySaveAreaOffsetInFrame(int checkedSaveAreaOffsetInFrame) {
         VMError.guarantee(saveAreaOffsetInFrame == checkedSaveAreaOffsetInFrame, "Must have a single value for the callee save register area");
     }
 
+    public boolean calleeSaveable(Register register) {
+        return offsetsInSaveArea.get(register) != null;
+    }
     public int getSaveAreaSize() {
```
Comment (olpaw) [inline substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/CalleeSavedRegisters.java:106]: Add a newline here. (see failing style gate)

Comment (adinn) [reply substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/CalleeSavedRegisters.java:107]: Fixed.

**Review by olpaw — APPROVED** [review]

**Comment by olpaw** [comment]
Running in internal CI now ...

**Comment by adinn** [comment]
Thanks Paul.

**Pull request merged**